src/components/ModalAddManyTokens.js

```diff
@@ -45,6 +45,7 @@ class ModalAddManyTokens extends React.Component {
   }
 
   componentDidMount = () => {
+    $('#addManyTokensModal').modal('show');
```

Comment:
The idea here is that the components won't need to use jquery directly, every time the component gets rendered it will show and when they are destroyed, they should hide

We also have a common prop `onClose` passed by the GlobalModal that should be called by the modals when the `hide.bs.modal` event is dispatched so the GlobalModal can properly remove it from the DOM